app/validators/policy_validator.rb

```diff
@@ -0,0 +1,13 @@
+class PolicyValidator < ActiveModel::EachValidator
+  def validate_each(record, attribute, value)
+    begin
+      policy = ApiClient.send(options[:method], value)
+    rescue GetIntoTeachingApiClient::ApiError => e
+      Rails.logger.error e # how do we handle a Bad Request ?
+    end
+
+    unless policy
+      record.errors[attribute] << (options[:message] || "is not included in the list")
```

Comment:
I believe this api is going away in Rails 6.1 ? I think it needs to be `record.errors.add`